lib/services/android-project-service.ts

```diff
@@ -111,7 +111,13 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 			}
 
 			// These files and directories should not be symlinked as CLI is modifying them and we'll change the original values as well.
-			this.copy(this.platformData.projectRoot, frameworkDir, "src", "-R");
+			if(pathToTemplate) {
+				let mainPath = path.join(this.platformData.projectRoot, "src", "main");
+				this.$fs.createDirectory(mainPath).wait();
+				shell.cp("-R", path.join(path.resolve(pathToTemplate), "*"), mainPath);
```

Comment:
Do we care that shell.cp can fail? We need to call shell.error to check for failures.